src/repr/src/row.rs

```diff
@@ -99,6 +101,42 @@ pub struct Row {
     data: SmallVec<[u8; Self::SIZE]>,
 }
 
+impl Arbitrary for Row {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Row>;
+
+    fn arbitrary_with(_: Self::Parameters) -> Self::Strategy {
+        let datums = prop_oneof![
+            Just(Datum::Null),
+            Just(Datum::False),
+            Just(Datum::Int16(0)),
+            Just(Datum::Int32(0)),
+            Just(Datum::Int64(0)),
+            Just(Datum::Float32(OrderedFloat(0.0))),
```

Comment:
I think we should add a TODO to allow more than one value for each Datum type. (Other than Datum::Null, Datum::False, etc.)

Reply:
I assume this comment will be adressed in #12125